src/pages/api/v1/images/index.ts

```diff
@@ -119,6 +119,7 @@ export default PublicEndpoint(async function handler(req: NextApiRequest, res: N
           },
           meta: image.meta,
           username: image.user.username,
+          baseModel: image.baseModel,
```

Comment:
We could allow filtering by `baseModel` in the public API as well, let's update the `imagesEndpointSchema` up above to allow the query string. Here's what it should be
```ts
const imageEndpointSchema = z.object({
  // ...
  baseModels: commaDelimitedEnumArray(z.enum(constants.baseModels).optional(),
});
```
That should do the trick

Reply:
Ah, missed this